data_steward/cdr_cleaner/cleaning_rules/base_cleaning_rule.py

```diff
@@ -276,6 +316,75 @@ def get_sandbox_dataset_id(self):
         """
         return self._sandbox_dataset_id
 
+    @property
+    def affected_tables(self):
+        """
+        Method to get tables that will be modified by the cleaning rule.
+        If getting the tables_affected dynamically logic needs to be
+        implemented in this method.
+        """
+        return self._affected_tables
+
+    @affected_tables.setter
+    def affected_tables(self, affected_tables):
+        """
+        Set the affected_tables for this class instance, raises error if affected_tables is not a list
+        """
+        if affected_tables:
+            if not type(affected_tables) is list:
```

Comment:
https://stackoverflow.com/questions/1549801/what-are-the-differences-between-type-and-isinstance
The python community generally prefers an instance check over a type check.
```
if not isinstance(affect_tables, list):
```
You can see that most argue we shouldn't use either, but in this case I think it is acceptable.  Because we want to set the variable to a good value that can be trusted and used later.